.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,23 @@
+name: Python application
```

Comment:
This is a too generic name, we are gonna have multiple actions so the name should be explicit, like "Test flake8:, and the yaml file should be renamed accordingly, like `flake.yml`